backend/core/src/main/java/com/feed_grabber/core/request/RequestRepository.java

```diff
@@ -24,5 +26,13 @@
 
     List<Request> findAllByQuestionnaireId(UUID id);
 
+    @Query("select r from Request r where r.closeDate is not NULL and r.questionnaire.company.id = :companyId")
+    List<Request> findAllReports(UUID companyId);
```

Comment:
Search all available reports for the company owner/HR.

Reply:
But why method called "findAllReports" returns list of request?